### PR TITLE
Update mParticle WEB SDK return types and parameters

### PR DESCRIPTION
**Pull request by alexs-mparticle**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.mparticle.com/developers/sdk/web/core-apidocs/classes/mParticle%20&%20mParticleInstance.html

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## mparticle__web-sdk ([<kbd>unpkg</kbd>](https://unpkg.com/browse/mparticle__web-sdk@latest/))
was missing the following properties:
1. Store
1. isIOS
1. config
1. configurePixel
1. generateHash

<details>
<summary>as well as these 40 other properties...</summary>
<p>addForwarder, wrap, isGeneratorFunction, mark, awrap, AsyncIterator, async, keys, values, ApplicationInformationOsEnum, ApplicationStateTransitionEventEventTypeEnum, ApplicationStateTransitionEventDataApplicationTransitionTypeEnum, BatchEnvironmentEnum, BreadcrumbEventEventTypeEnum, CommerceEventEventTypeEnum, CommerceEventDataCustomEventTypeEnum, CrashReportEventEventTypeEnum, CustomEventEventTypeEnum, CustomEventDataCustomEventTypeEnum, DeviceCurrentStateDeviceOrientationEnum, DeviceCurrentStateStatusBarOrientationEnum, DeviceInformationPlatformEnum, EventTypeEnum, IdentityTypeEnum, NetworkPerformanceEventEventTypeEnum, OptOutEventEnum, ProductActionActionEnum, ProfileEventEventTypeEnum, ProfileEventDataProfileEventTypeEnum, PromotionActionActionEnum, PushMessageEventEventTypeEnum, PushMessageEventDataPushMessageTypeEnum, PushMessageEventDataApplicationStateEnum, PushMessageEventDataPushMessageBehaviorEnum, PushRegistrationEventEventTypeEnum, SessionEndEventEventTypeEnum, SessionStartEventEventTypeEnum, SourceInformationChannelEnum, UserAttributeChangeEventEventTypeEnum, UserIdentityChangeEventEventTypeEnum</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 00b8868f16d78cd5433782afa219d6416dcee945
</p>

File: types/mparticle__web-sdk/index.d.ts [hunk 1]
```diff
@@ -512,7 +484,8 @@ export interface IdentifyRequest {
 
 export type MPID = string;
 export interface User {
-    getUserIdentities: () => UserIdentities;
+    getUserIdentities: () => IdentityApiData;
+    // getUserIdentities: () => UserIdentities;
```
Comment (rmi22186) [inline types/mparticle__web-sdk/index.d.ts:488]: @alexs-mparticle - this commented former code should be removed

**Review by rmi22186 — CHANGES_REQUESTED** [review]
Just the one comment removal, then we are good to merge

**Review by rmi22186 — APPROVED** [review]

**Comment by rmi22186** [comment]
Ready to merge

**Pull request merged**